src/coreclr-debug/install.ts

```diff
@@ -2,12 +2,10 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-'use strict';
 
 import { CoreClrDebugUtil } from './util';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as fs_extra from 'fs-extra-promise';
```

Comment:
not quite, looks like this is used in assets.ts and features/commands.ts

---
In reply to: [86635042](https://github.com/OmniSharp/omnisharp-vscode/pull/873#discussion_r86635042) [](ancestors = 86635042)